recipes/galario/meta.yaml

```diff
@@ -0,0 +1,92 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "galario" %}
+{% set version = "1.0.1" %}
+{% set github = "https://github.com/mtazzari/galario" %}
+{% set sha256 = "986ea0880f8066cb64251351ad40720b080c2cfe78ab3b5dc347564d0a858979" %}
+# sha256 is the prefered checksum -- you can get it for a file with either of
+#  `openssl sha256 <file name>`
+#  `sha256sum <file name>`
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # for a release with tarball
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: {{ github }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  # from the master branch: only for testing
+  # git_url: {{ github }}.git
+  # git_url: https://github.com/fredRos/galario.git
+  # git_rev: master
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [win]
+
+requirements:
+  build:
+    # compilation
+    - cmake
+    - fftw
+    - gcc  # [osx]
+    # python bindings
+    - python
+    - cython
+    - numpy 1.11.*
+    - pytest
```

Comment:
Numpy is needed because we have to link to its C API via cython and pytest is needed in our cmake setup such that library paths are set correctly to be able to run the tests in the build directory. This is a quite a pain to get right, so we use a module from [here](https://github.com/UCL/GreatCMakeCookOff) which requires `pytest` at build time